core-bundle/src/Command/LintServiceIdsCommand.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Yaml\Yaml;
+use Webmozart\PathUtil\Path;
+
+class LintServiceIdsCommand extends Command
+{
+    protected static $defaultName = 'contao:lint-service-ids';
+
+    private static array $ignore = [
+        'listener',
+        'migration',
+        'picker',
+        'runtime',
+        'subscriber',
+    ];
+
+    private static array $ignoredChunks = [
+        'authentication',
+        'candidates',
+        'data_collector',
+        'http_kernel',
+        'insert_tag',
+        'logout',
+        'matcher',
+        'remember_me',
+        'schema',
+        'voter',
+    ];
+
+    private static array $exceptions = [
+        'contao.assets.assets_context',
+        'contao.assets.files_context',
+        'contao.csrf.token_storage',
+        'contao.fragment.forward_renderer',
+        'contao.image.deferred_image_storage',
+        'contao.listener.element_template_options',
+        'contao.listener.module_template_options',
+        'contao.migration.version400.version400_update',
+        'contao.monolog.handler',
+        'contao.monolog.processor',
+        'contao.resource_finder',
+        'contao.response_context.accessor',
+        'contao.response_context.factory',
+        'contao.routing.candidates',
+        'contao.routing.input_enhancer',
+        'contao.routing.page_registry',
+        'contao.security.authentication_listener',
+        'contao.security.authentication_provider',
+        'contao.security.backend_user_provider',
+        'contao.security.entry_point',
+        'contao.security.frontend_user_provider',
+        'contao.security.token_checker',
+        'contao.security.user_checker',
+        'contao.session.contao_backend',
+        'contao.session.contao_frontend',
+    ];
+
+    private string $projectDir;
+
+    public function __construct(string $projectDir)
+    {
+        $this->projectDir = $projectDir;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Checks the Contao service IDs.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $files = Finder::create()
+            ->files()
+            ->name('*.yml')
+            ->path('-bundle/src/Resources/config')
+            ->in(Path::join($this->projectDir, 'vendor/contao/contao'))
+        ;
+
+        $tc = 0;
```

Comment:
What does `tc` and `fc` stand for? :see_no_evil:

Reply:
I guess `totalCount` and `fileCount` 🙃